src/styles/base.scss

```diff
@@ -412,14 +503,21 @@ ul {
   border-radius: 20px;
   margin-right: 10px;
   margin-top: 5px;
-  margin-bottom: 0 !important;
+  margin-bottom: 5px !important;
```

Comment:
@Sanji515 did you check it for smaller devices?

Reply:
@lunayach yeah I've checked I think it's looking good, but please check on your side too and let me know if any changes needed